### PR TITLE
BUGFIX: missing import statement in __init__.py

### PR DESCRIPTION
**Pull request by Yaxuan-Lii**
This is a copy of #34. Because of the wrong branch name.

### CODE REVIEW

**Comment by ajjackson** [comment]
This looks promising, the tests failed due to ImportError as expected. Now it can be fixed and we willl be a bit more confident that the fix did something!

**Comment by ajjackson** [comment]
The last commit "fix the imports" seems to add a lot of unnecessary files to the project. Probably they were added accidentally. Can you clean that up?

**Review by Yaxuan-Lii — COMMENTED** [review]
But fail to delete them.

File: test/test_process_pdos.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import numpy as np
+import galore
+import unittest
+from numpy.testing import assert_array_almost_equal
+
+
+
+class test_process_pdos(unittest.TestCase):
+
+    def test_lorentzian(self):
```
Comment (ajjackson) [inline test/test_process_pdos.py:11]: test_lorentzian and test_broaden don't test `process_pdos` so I don't think they belong in this class?

File: test/test_process_pdos.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        xmin, xmax = (-10, 4)
+        weighting='Alka'
+
+        plotting_data = galore.process_pdos(input=[vasprun],
+                                    gaussian=0.3, lorentzian=0.2,
+                                    xmin=xmin, xmax=xmax,
+                                    weighting=weighting)
+        self.assertEqual(plotting_data['O']['energy'][0], -10.0)
+
+
```
Comment (ajjackson) [inline test/test_process_pdos.py:37]: There are Python conventions for how many blank lines should be between sections; try running `flake8` on this file.

**Review by ajjackson — CHANGES_REQUESTED** [review]
I'll clean up the missing files tonight, here are a couple of other things to look at.

File: test/test_process_pdos.py (hunk 1 above)
Comment (ajjackson) [reply test/test_process_pdos.py:8]: Still, they appear to be new tests and it's good to test them _somewhere_. Really the test suite could use a bit of modernisation (Pytest maybe?) So for now let's keep them.

**Review by ajjackson — APPROVED** [review]

**Pull request merged**